x/burn/expected/expected.go

```diff
@@ -0,0 +1,15 @@
+package expected
```

Comment:
This is a PR that was applied to the previous version, so it has nothing to do with this PR.